sphinx_gallery/utils.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities
+=========
+
+Miscellaneous utilities.
+"""
+# Author: Eric Larson
+# License: 3-clause BSD
+
+from __future__ import division, absolute_import, print_function
+
+import tempfile
+from shutil import rmtree
+
+
+class _TempDir(str):
```

Comment:
If this is to be used for tests, you could use the [`tmpdir` fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmpdir-fixture).

Reply:
Neat, didn't know about that